src/DataTypes.ts

```diff
@@ -19,3 +23,19 @@ export interface MarkerFileData {
     frames: Array<MarkerFrame>;
 }
 
+// -------------------- forces ---------------------
+
+export interface Force {
+    position: Point3D;   //px, py, pz
+    components: Point3D; //vx, vy, vz
+    torque: number;
```

Comment:
Based on the file it looks like torque has multiple components?

Reply:
The columns are labeled `ground_torque_x`, `ground_torque_y`, `ground_torque_z`, but only torque about the y-axis shows data, presumably because force plates (on the ground) only record torque about the vertical (I'm not sure what else would make sense in the context of a foot in a twisting motion). I thought it might be better to only capture that component to reduce clutter.

Reply:
Thoughts?

Reply:
fair enough, could always be changed if a new use case comes up